### PR TITLE
[TACHYON-1428] Documented the tachyon.replay, tachyon.retry and tachyon.security packages in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: common/src/main/java/tachyon/retry/ExponentialBackoffRetry.java [hunk 1]
```diff
@@ -29,6 +29,13 @@
   private final int mBaseSleepTimeMs;
   private final int mMaxSleepMs;
 
+  /**
+   * Constructs a new retry which grows exponentially.
```
Comment (aaudiber) [inline common/src/main/java/tachyon/retry/ExponentialBackoffRetry.java]: Constructs a new retry facility which sleeps for an exponentially increasing amount of time between retries.

**Comment by aaudiber** [comment]
Thanks for the improvement, overall LGTM. I left one minor comment

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comment. I updated the PR accordingly.

**Comment by aaudiber** [comment]
Great, LGTM.

**Pull request merged**